tests/e2e/catalog.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { expect, test } from "./fixtures";
+
+test.describe("Catalog page", () => {
+  test("view catalog page header", async ({ authenticatedPage }) => {
+    await authenticatedPage.goto("/catalog");
+    await expect(
+      authenticatedPage.getByRole("heading", { name: "MCP Server Catalog" }),
+    ).toBeVisible();
```

Comment:
can we test the mcp is visible in the page?

Reply:
sure! to keep things simple for this initial PR I just added simple text assertions for that